packages/snaps-sdk/src/jsx/components/form/DropdownOption.ts

```diff
@@ -0,0 +1,41 @@
+import { createSnapComponent } from '../../component';
+
+/**
+ * The props of the {@link Dropdown} component.
+ *
+ * @property value - The value of the dropdown option. This is used to populate the
+ * state in the form data.
+ */
+type DropdownOptionProps = {
+  value: string;
+  children: string;
+};
+
+const TYPE = 'DropdownOption';
+
+/**
+ * A dropdown option component, which is used to create a dropdown option. This component
+ * can only be used as a child of the {@link Dropdown} component.
+ *
+ * @param props - The props of the component.
+ * @param props.value - The value of the dropdown option. This is used to populate the
+ * state in the form data.
+ * @returns A dropdown option element.
+ * @example
+ * <Dropdown name="dropdown">
+ *  <DropdownOption value="option1">Option 1</DropdownOption>
+ *  <DropdownOption value="option2">Option 2</DropdownOption>
+ *  <DropdownOption value="option3">Option 3</DropdownOption>
+ * </Dropdown>
+ */
+export const DropdownOption = createSnapComponent<
```

Comment:
Should we just name this `Option`, like HTML's `<select>` and `<option>`? This seems cleaner:

```tsx
<Dropdown name="dropdown">
  <Option value="option1">Option 1</DropdownOption>
  <Option value="option2">Option 2</DropdownOption>
  <Option value="option3">Option 3</DropdownOption>
</Dropdown>
```

Reply:
I was thinking that too, just curious if it is obvious enough 😄

Reply:
Obvious enough in what sense? It's a direct child of `Dropdown` so I think it should be clear what it does, and it's consistent with what I've been trying to do with other components (i.e., `Field` instead of `FormField`).